test/import/importConfiguration.test.ts

```diff
@@ -1729,14 +1723,20 @@ describe('importConfiguration', () => {
       config = importConfiguration(minYAML, 'test-config.yaml');
       expect(config.parameters[0]).toEqual({ code: 'copyrightyear', value: '2020' });
     });
-    it('should report an error if copyrightYear/copyrightyear is missing', () => {
+    it('should report an error if copyrightYear/copyrightyear is missing and FSHOnly is false/unset', () => {
```

Comment:
Similarly, should this just say "false" now?

Reply:
Yes it should.